### PR TITLE
fix: sanitize file names

### LINKED ISSUE #1238: Sanitize file names
There is a syntax issue [here](https://github.com/googleapis/googleapis-gen/blob/master/google/cloud/run/v2/run-v2-py/google/cloud/run_v2/services/revisions/async_client.py#L39) with the [generated client](https://github.com/googleapis/googleapis-gen/tree/master/google/cloud/run/v2/run-v2-py) for Cloud Run. The issue is that there is a proto file [here](https://github.com/googleapis/googleapis/blob/master/google/cloud/run/v2/k8s.min.proto) which does not use the correct snake case format.

We should fix this issue in the generator as it may be difficult to rename the proto. I've opened a [PR](https://github.com/googleapis/api-linter/pull/948) to improve the api-linter to help catch the issue upstream in the future.

To workaround the issue, the line `from google.cloud.run_v2.types import k8s.min` should be changed to `from google.cloud.run_v2.types import k8s_min`.

### PR DESCRIPTION
**Pull request by parthea**
Fixes: #1238

### CODE REVIEW

**Review by busunkim96 — APPROVED** [review]
@atulep Could you also take a look?

**Comment by parthea** [comment]
Hi @atulep, please could you review/approve?

**Review by atulep — COMMENTED** [review]
I left comments yesterday, but I didn't realize my review was "pending". Sorry for delay.

File: tests/unit/schema/test_api.py [hunk 1]
```diff
@@ -209,6 +209,23 @@ def test_proto_names():
     assert proto.disambiguate('foo') == '_foo'
 
 
+def test_proto_with_invalid_characters():
```
[the remark this answers is not in the document]
Comment (parthea) [reply tests/unit/schema/test_api.py:212]: Done

**Comment by parthea** [comment]
All review comments have been addressed. Merging.

**Pull request merged**